provider/datagen/tests/data/json/units/info@1/und.json

```diff
@@ -0,0 +1,4150 @@
+{
+  "units_info": {
+    "100-kilometer": {
+      "dimension": null,
+      "convert_info": 0
+    },
+    "acre": {
+      "dimension": null,
```

Comment:
why are there so many `"dimension": null`s here? Most of these are not dimensionless, `acre` for example has dimension `area`.

Reply:
If the dimension is null, this means you know the dimension when you derived them.

For example: `acre` --> `square-meter` --> the dimension is `area`

Reply:
That's an extra step. We can store the dimension here for free and avoid the base unit lookup.

Reply:
this is a good question, there are infinite number of units under the same dimension.

For example: `square-centimeter`, `square-millimeter` , `square-foot` ... etc. All of them are labeled as `area`.

Therefore, we have always to derive.

Actually, if the users do not need to know the `dimension`, it should be deleted.

I have updated the issue.

Reply:
There's a single dimension per unit. You can just store it here. I have also updated the issue.

Reply:
I'm willing to spend an extra hop on lookup if it reduces data size. If pulling the dimensions directly into this table doesn't adversely impact data size, though, it seems like a good idea.